chia/_tests/core/data_layer/test_data_store.py

```diff
@@ -1304,37 +1344,55 @@ async def test_data_server_files(
 
         keys: list[bytes] = []
         counter = 0
-
-        for batch in range(num_batches):
-            changelist: list[dict[str, Any]] = []
-            for operation in range(num_ops_per_batch):
-                if random.randint(0, 4) > 0 or len(keys) == 0:
-                    key = counter.to_bytes(4, byteorder="big")
-                    value = (2 * counter).to_bytes(4, byteorder="big")
-                    keys.append(key)
-                    changelist.append({"action": "insert", "key": key, "value": value})
+        num_repeats = 2
+
+        for _ in range(num_repeats):
```

Comment:
Was this an experiment or does it trigger more coverage somehow?  Do we want to keep it?